readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://vk-workshop.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://vk-workshop.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="965" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/dfb358ad-e658-473d-a588-011bc44b98b4">